notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOMTN-092_m1m3_ics_historical_analysis.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>I don't see the same things in the correlation map that I get.</p><p>I see the following:</p><ul><li>measuredForceMin1 is correlated with measuredForceMin3 all the other are anti-correlated</li><li>measuredForceMax1 is correlated with measuredForcedMin1 and Min4</li></ul><p>I also don't understand how do we see that Min1 and Min4 are flipped ?</p><p>But may be I am missing something</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
The analysis seems to depend on the dayObs. This makes sense since different days have different behavior initially. They correspond to other software versions. With that said, I think that, it this part of the analysis is really useful, we should create a new PR to do it. Do you agree or do you prefer to have the analysis here?

Reply:
If this depends on the dayObs, I agree that it would require a separate analysis